engines/grim/lua.cpp

```diff
@@ -83,6 +83,14 @@ void LuaObjects::addNil() {
 	_objects.push_back(obj);
 }
 
+void LuaObjects::add(const float *ar, int len) {
```

Comment:
changes to Lua will not be accepted. patches will needed to be modified to use unmodified Lua core

Reply:
but, this is something for later in this PR, let's focus on other stuff first

Reply:
This is used in hotspot.cpp.

Reply:
ah, never mind, I didn't look closely, I thought it was Lua core